core/templates/core/homepage-feature.html

```diff
@@ -1,16 +1,18 @@
-{% load wagtailcore_tags static %}
+{% load static core_tags wagtailcore_tags wagtailimages_tags %}
 
 <div class="jumbotron feature" style="height: 400px;">
 
-    <div class="feature-image" style="background-image: url('{% static 'img/assets/diplomacy-unbound.png' %}')">
+    {% image self.typed_featured_item.main_image original as background_image %}
+    <div class="feature-image" style="background-image: url(' {{ background_image.url }} ')">
         <div class="feature-image-overlay"></div>
     </div>
+
     <div class="feature-text">
         <h1>
-        <a style="color: {{ self.featured_item.font_style.text_colour }}; font-size: {{ self.featured_item.font_style.font_size }}em; line-height: {{ self.featured_item.font_style.line_size }}%; "href="{{ self.featured_item.url }}">{{ self.featured_item.title }}</a></h1>
-        <div class="feature-line" style="color: {{ self.featured_item.font_style.text_colour }};">{{ self.featured_item.articlepage.subtitle|richtext }}</div>
+        <a style="color: {{ self.typed_featured_item.font_style.text_colour.rgb_value }}; font-size: {{ self.typed_featured_item.font_style.font_size }}em; line-height: {{ self.typed_featured_item.font_style.line_size }}%; "href="{{ self.featured_item.articlepage.url }}">{{ self.featured_item.title }}</a></h1>
+        <div class="feature-line" style="color: {{ self.typed_featured_item.font_style.text_colour.rgb_value }};">{{ self.typed_featured_item.articlepage.excerpt|richtext }}</div>
```

Comment:
Don't need to use .articlepage with self.typed_featured_item.   

Should check if there is an excerpt before using it.  Since InDepths don't have excerpts.  Perhaps use subtitle if it exists, since InDepths do have subtitles.